initrd/etc/functions

```diff
@@ -81,6 +81,36 @@ confirm_totp()
 	echo
 }
 
+reseal_tpm_disk_decryption_key()
+{
+	TRACE "Under /etc/functions:reseal_tpm_disk_decryption_key"
+	#Can be called only on resealing, not on tpm reset:
```

Comment:
But TPM reset is calling it, should that be removed?  (gui-init line 534)

Reply:
Comment deleted in 8809588a, so assuming it works for TPM reset as well, looks good :+1: